src/molecules/Select/Input.tsx

```diff
@@ -200,6 +206,7 @@ const Input: React.SFC<Props> = ({
     <Wrapper
       id={inputProps.id}
       readOnly={inputProps.readOnly}
+      tabIndex={0}
```

Comment:
If you want the optional prop to be used, you can destructure the `tabIndex` prop and use it here. Otherwise the divs will always have tabIndex="0", which I don't think will do exactly what we want accessibility-wise.